security/keycloak-authz/src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ quarkus.oidc.credentials.secret=test-application-client-secret
 quarkus.oidc.token.lifespan-grace=60
 
 quarkus.keycloak.policy-enforcer.enable=true
-quarkus.keycloak.policy-enforcer.paths.health.path=/health/*
+quarkus.keycloak.policy-enforcer.paths.health.path=/q/health/*
```

Comment:
There should be "auto-redirect" for such cases.

By auto-redirect I mean there should be HTTP 301 Moved Permanently response

Reply:
https://github.com/quarkusio/quarkus/blob/master/integration-tests/main/src/test/java/io/quarkus/it/main/HealthTestCase.java#L59

Reply:
This ca hit our customers :/ We should create an issue for this

Reply:
```
curl -vvv http://127.0.0.1:8080/health
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to 127.0.0.1 (127.0.0.1) port 8080 (#0)
> GET /health HTTP/1.1
> Host: 127.0.0.1:8080
> User-Agent: curl/7.64.1
> Accept: */*
>
< HTTP/1.1 301 Moved Permanently
< location: http://127.0.0.1:8080/q/health
< content-length: 0
<
* Connection #0 to host 127.0.0.1 left intact
* Closing connection 0
```

Reply:
[Github issue](https://github.com/quarkusio/quarkus/issues/14619)